### ISSUE TITLE
fix: UI Fix's in the main website

### ISSUE THREAD

**Issue by goswamianshuman** [Shuffle/Shuffle#1238]
**Describe the bug**
I was going through the shuffle website: https://shuffler.io/ and found some UI issues which I think are necessary to fix.

**To Reproduce**
1. get to https://shuffler.io/
2. scroll down
3. now check the screenshots which are pasted below:

**Screenshots**
1) when we click on creators there is an overflow effect:
![shuffle-one](https://github.com/Shuffle/Shuffle/assets/68741416/b868e6f2-b7cc-4886-9870-ca22bf440ebf)

2) cards in the workflow section can be a little better:
![image](https://github.com/Shuffle/Shuffle/assets/68741416/fd567745-43e7-4488-82af-f0c8c28623ac)

3) when we hover on cards in this Apps section there is a student effect while hovering on each card also each card has an uneven size while hovering:
![image](https://github.com/Shuffle/Shuffle/assets/68741416/056bfb71-420a-4bfe-9fa4-a5b307401464)

4) In this section the cards can be improved by fixing image position and text size: 
![image](https://github.com/Shuffle/Shuffle/assets/68741416/93a9ea08-c8d1-4c11-9365-1c3fd5413cc2)

5) contact form can also improved
![image](https://github.com/Shuffle/Shuffle/assets/68741416/aabb5679-f771-430d-a793-ae539eff0c82)

6) In this newsletter section button position is not correct and we can create a cool newsletter section:
![image](https://github.com/Shuffle/Shuffle/assets/68741416/3ef61858-e891-4632-bea7-52782fab48f2)

#### expected behavior
I think this UI elements must be / can be fixed as soon as possible so that the user can have a good UI experience.

**Comment by goswamianshuman** [comment]
## I would like to work on this issue, I think we can do much better things here. please assign me these frontend tasks, if the designs and fixes are not good you can re-assign this issue.

**Comment by frikky** [comment]
> @frikky I would like to work on this issue, I think we can do much better things here. please assign me these frontend tasks, if the designs and fixes are not good you can re-assign this issue 😄.

Hey there! 

We could indeed use some help making the website look more professional, and have some more flare. If you'd like, we can set up a call and we can help you get set up to play around with these UI's. 

Would love to see some design from your side as well if that's your thing 👍

**Comment by goswamianshuman** [comment]
> > @frikky I would like to work on this issue, I think we can do much better things here. please assign me these frontend tasks, if the designs and fixes are not good you can re-assign this issue 😄.
> 
> Hey there!
> 
> We could indeed use some help making the website look more professional, and have some more flare. If you'd like, we can set up a call and we can help you get set up to play around with these UI's.
> 
> Would love to see some design from your side as well if that's your thing 👍

yes sure @frikky I was waiting for some similar response 😄. please provide me some time so that I can create an updated overlook of updated UI.

**Comment by frikky** [comment]
> > > @frikky I would like to work on this issue, I think we can do much better things here. please assign me these frontend tasks, if the designs and fixes are not good you can re-assign this issue 😄.
> > 
> > 
> > Hey there!
> > We could indeed use some help making the website look more professional, and have some more flare. If you'd like, we can set up a call and we can help you get set up to play around with these UI's.
> > Would love to see some design from your side as well if that's your thing 👍
> 
> yes sure @frikky I was waiting for some similar response 😄. please provide me some time so that I can create an updated overlook of UI.

Just to not waste your time: We already did a lot of work on the Landing page itself that is not released, so focusing on each individual element (like Contact form and the Tabs for the search engine) would be great places to start

**Comment by goswamianshuman** [comment]
> > > > @frikky I would like to work on this issue, I think we can do much better things here. please assign me these frontend tasks, if the designs and fixes are not good you can re-assign this issue 😄.
> > > 
> > > 
> > > Hey there!
> > > We could indeed use some help making the website look more professional, and have some more flare. If you'd like, we can set up a call and we can help you get set up to play around with these UI's.
> > > Would love to see some design from your side as well if that's your thing 👍
> > 
> > 
> > yes sure @frikky I was waiting for some similar response 😄. please provide me some time so that I can create an updated overlook of UI.
> 
> Just to not waste your time: We already did a lot of work on the Landing page itself that is not released, so focusing on each individual element (like Contact form and the Tabs for the search engine) would be great places to start

okay sure then I can start the working please let me know when can have a call.

**Comment by goswamianshuman** [comment]
here is my email: anshumangoswami07@gmail.com you can share the details here.

**Comment by goswamianshuman** [comment]
Hey @frikky, check this https://www.figma.com/file/tYLvQYTB2ENrN4yNmZcBqK/shuffle?type=design&node-id=0%3A1&mode=design&t=LpXnusxsYA0WLEgI-1 

I created them yesterday only but got a big internet issue from my side and was unable to work for a long time and in the morning went for my college exam. please have a look if you consider them good then mark new elements in the comments I'll try to create them as well and let me know when I can code them too.

**Comment by frikky** [comment]
> Hey @frikky, check this https://www.figma.com/file/tYLvQYTB2ENrN4yNmZcBqK/shuffle?type=design&node-id=0%3A1&mode=design&t=LpXnusxsYA0WLEgI-1
> 
> I created them yesterday only but got a big internet issue from my side and was unable to work for a long time and in the morning went for my college exam. please have a look if you consider them good then mark new elements in the comments I'll try to create them as well and let me know when I can code them too.

Getting our lead designer @iiAnna to have a look at this :)

Short from me: I like the ideas, but am unsure if too much color will be a good thing or not for these components. Awaiting @iiAnna :)

**Comment by goswamianshuman** [comment]
yes sure, I know designs might differ as I haven't seen the landing page which is already designed but waiting for the response 😄.

**Comment by iiAnna** [comment]
Regarding the rest of the post , we are working on improving those issues. Thank you for your feedback and let us know what else we can improve on!

**Comment by goswamianshuman** [comment]
> Hey, @goswamianshuman I like the overall design ideas. I think there are some very nice touches we can learn from and the look and feel is very clean. There are some things that can be improved. The overall design needs to match our new Style Guides to keep it consistent throughout the platform. Also, when using bright colors they need to be used in moderation to not distract from different elements in the page. Considering we have a dark theme is it something to keep in mind. For example, in the search bar design, I really like the idea of having the tags but the color is too vibrant and can be distracting, as well as the outline around the input. Usually, an input gets a border color when typing. Also, wording matters a lot in CTA's to indicate what action the user is going to take. Therefore, "signup" does not match the action but "search" does. Those are the main points. I am very thankful for you giving us feedback it is always appreciated! Let me know what you think, would love to hear from you again. :)

Hey @iiAnna I got your point, and I apologize for the CTA bug mistakenly I just inserted signup instead of search. I would surely like to work on UI if its acceptable (referring to code the UI design). if you feel we can make changes an should look forward in developing the UI will be a great opportunity.🙂

**Comment by goswamianshuman** [comment]
> Regarding the rest of the post , we are working on improving those issues. Thank you for your feedback and let us know what else we can improve on!

yes looking forward with other issues too...😄

**Comment by iiAnna** [comment]
@goswamianshuman The designs are definitely going in the right direction! No worries about the CTA, happens. :)

**Comment by goswamianshuman** [comment]
@iiAnna @frikky so, what's next? am I going to work on it? 🤔

**Comment by frikky** [comment]
> @iiAnna @frikky so, what's next? am I going to work on it? 🤔

Sure! The search engine could for sure need some help, so I'd suggest starting there. Remember to take @iiAnna's suggestions into account

**Comment by goswamianshuman** [comment]
> > @iiAnna @frikky so, what's next? am I going to work on it? 🤔
> 
> Sure! The search engine could for sure need some help, so I'd suggest starting there. Remember to take @iiAnna's suggestions into account 

Great! ✨

**Comment by goswamianshuman** [comment]
> > @iiAnna @frikky so, what's next? am I going to work on it? 🤔
> 
> Sure! The search engine could for sure need some help, so I'd suggest starting there. Remember to take @iiAnna's suggestions into account 

Hey @iiAnna should I add you in the project by including your email? 🤔

**Comment by iiAnna** [comment]
@goswamianshuman Yes, please send me the link via direct message when you can.

Have a good weekend!

**Comment by goswamianshuman** [comment]
@iiAnna on GitHub there is no direct message feature. can you please share your email or shall I just make permission to be publically available?

**Comment by Monilprajapati** [comment]
I have read the entire conversation between @frikky, @goswamianshuman, and @iiAnna. I am interested in contributing to the development of those designs in coordination with @goswamianshuman. will be in touch with him for the next steps, and I'm excited to get involved in refining the designs. @iiAnna Your insights on design consistency and the use of colors make a lot of sense. I'm ready to collaborate and make these changes a reality. What do you think? @frikky

**Comment by Monilprajapati** [comment]
> @iiAnna on GitHub there is no direct message feature, can you please text me at [anshumangoswami07@gmail.com](mailto:anshumangoswami07@gmail.com) or shall I just make permission to be publically available?

Can you share your discord id so that i can ping you there!

**Comment by goswamianshuman** [comment]
> > @iiAnna on GitHub there is no direct message feature, can you please text me at [anshumangoswami07@gmail.com](mailto:anshumangoswami07@gmail.com) or shall I just make permission to be publically available?
> 
> Can you share your discord id so that i can ping you there!

it's the same as my GitHub username: goswamianshuman

**Comment by iiAnna** [comment]
Hey, you can find me on the Shuffle discord, my id is iiFork

**Comment by goswamianshuman** [comment]
yes sure.

> Hey, you can find me on the Shuffle discord, my id is iiFork
